test/integration/org_admin_js_test.rb

```diff
@@ -142,7 +142,7 @@ def test_org_admins_can_work_with_resources_in_their_organization
       visit domains_path
       wait_for_ajax
 
-      click_link 'Create Domain'
+      click_button 'Create Domain'
```

Comment:
I think there should be at least one domain in the db so these should stay `click_link`, it's the link on the top of the domain table and not the welcome page? (same for the others below)
Previously only one test was failing, that is the one we need to fix since it is the one that goes to the welcome page (i don't remember which one it was though)